clang/lib/DPCT/ExprAnalysis.cpp

```diff
@@ -813,6 +814,30 @@ void ExprAnalysis::analyzeExpr(const ExplicitCastExpr *Cast) {
   dispatch(Cast->getSubExprAsWritten());
 }
 
+bool isSpecialFunctions(const CallExpr *CE, const std::string &RefString) {
```

Comment:
check name is min/max, and check if this function can be evaluated during migration

Reply:
done